### ISSUE TITLE
SVG output for bamFingerprint via file ext.

### ISSUE THREAD

**Issue by mpi-pebert** [fidelram/deepTools#75]
The `bamFingerprint` tool does not produce SVG output based on the file extension `.svg` only (i.e. simply calling `bamFingerprint --plotFile foofile.svg`); it appears as if the setting for the output file format is overwritten somewhere, since calling `bamFingerprint --plotFile foofile.svg --plotFileFormat svg` does generate an SVG output plot.
I am using deepTools 1.5.8.1 and matplotlib 1.3.1. Notably, setting SVG output via the file extension works for other deepTools, e.g., `bamCorrelate --plotFile foofile.svg` produces an SVG output file.

**Comment by fidelram** [comment]
I could not reproduce this behaviour in the master branch.

**Comment by mpi-pebert** [comment]
I can confirm the bug again for version 1.5.9.1
I tested with the following call: `bamFingerprint -p 6 --plotFile foofile.svg --bamfiles Input.bwa.20131017.bam H3K4me3.bwa.20131016.bam --labels Input H3K4me3 --fragmentLength 200` and the result `foofile.svg` is a binary file starting with `<89>PNG`. When I open the file in Gimp, it also tells me that this is a `PNG image`.

**Comment by fidelram** [comment]
the only difference then is the matplotlib version. I am using 1.4.3.

I compared the code from the cases that work and from bamFingerprint but I did not see any difference. 

The following method to install deeptools is currently working well for me:

Download and install the anaconda package (https://store.continuum.io/cshop/anaconda/). This includes the latest versions of scipy numpy and matplotlib that are always difficult to install.

Then using the pip version from anaconda simply type:

pip install git+https://github.com/fidelram/deepTools

This should take care to install bx-python and pysam
